### ISSUE TITLE
Proxy list import

### ISSUE THREAD

**Issue by Dramorian** [ramjke/Translumo#9]
Apparently, at the moment we have to enter them manually.
Any plans to implement a list import?

Though it's been said that 1-2 should be enough, the input form is poorly made and doesn't allow copypaste.

**Comment by Danily07** [comment]
Yep, I understand that current proxy input has quite rough implementation. I defenitely have plans to improve it and then consider option with import from file. It would be useful, thanks!

**Comment by ramjke** [comment]
Removed in 1.1.0
